admin/class-yoast-form.php

```diff
@@ -545,4 +547,25 @@ public function toggle_switch( $var, $values, $label ) {
 
 		echo '<a></a></div></fieldset><div class="clear"></div></div>' . "\n\n";
 	}
+
+	/**
+	 * Check if the user is on a page of Yoast SEO
+	 *
+	 * @return bool
+	 */
+	protected function load_i18n_promo_class() {
+		$wpseo_admin = new WPSEO_Admin();
+
+		$current_screen = get_current_screen()->id;
```

Comment:
There is a `WPSEO_Utils::is_yoast_seo_page()` method that can handle the check if the current page is a Yoast SEO page. This would make lines 559 - 563, redundant.